### ISSUE TITLE
Links in {{%alert}} don't resolve from footer href

### ISSUE THREAD

**Issue by kingdonb** [fluxcd/website#161]
We often build links in this style:

```
A paragraph about something [that needs a link]

[that needs a link]: http://foobar.com
```

This doesn't apparently work together with the {{%alert}} notation in Hugo

Examples:

https://fluxcd.io/docs/use-cases/gh-actions-manifest-generation/

> When migrating to Flux v2
> Users will find that [some guidance has changed since Flux v1]. Tagging images with a GIT_SHA was a common practice that is no longer supported by Flux’s Image Automation. A newer alternative is adding timestamp or build number in [Sortable image tags], preferred by the image-automation-controller.

The links don't resolve their references from the footer when they are in an alert, I guess?

It works if you move the link reference inline like this

```
{{% alert title="Additional options" %}}
A paragraph about something [that needs a link](http://foobar.com)
```

I was going to go through and start looking for instances of this problem in other documents, (but maybe there's an option to re-enable this behavior globally, or an open issue at hugo if it's not meant to be disabled by default.)

**Comment by dholbach** [comment]
I could imagine it needs fixing like so: https://github.com/fluxcd/website/blame/main/content/en/docs/migration/flux-v1-migration.md#L13-L16

**Comment by kingdonb** [comment]
That fixed it (merging #162 resolved the issue in my doc, at least).

I haven't completed a review of other docs, so I'll leave this issue open and come back to it sooner if it turns out to be urgent.
